contrib/icons/Makefile

```diff
@@ -1,10 +1,15 @@
 .PHONY: help clean favicons
 
 help:
-	@echo "Re-generate icons in platform specific formats."
-	@echo
-	@echo "Available targets:"
-	@echo "  hicolor             - Freedesktop icon directory structure"
+	@echo "Re-generate icons in platform specific formats or install pre-rendered png icons"
+	@echo ""
+	@echo "Usage:"
+	@echo "  make install             # install icons to \$${DESTDIR}/usr/local/share/icons/..."
+	@echo "  make install prefix=/usr # install icons to \$${DESTDIR}/usr/share/icons/..."
+	@echo "  make install datadir=foo # install icons to \$${DESTDIR}foo/icons/..."
+	@echo ""
```

Comment:
The general case would be nice to show, as it makes the relationship between PREFIX and DATA very clear (without having to deduce it through the two examples).

`make install [prefix=PREFIX] [datadir=DATADIR] # installs to \$${DESTDIR}/PREFIX/DATADIR/icons/...`

However its a tad lengthy.. maybe wrap it?

Reply:
But it does not install to `$DESTDIR/prefix/datadir`… - that would result in packagers being cut by placing files in `/usr/usr/share/icons/...`.

`prefix` is for replacing `/usr/local` with e.g. `/usr` - for distributors/distributions/packagers who prefer to use automake prefix syntax.

`datadir` is for replacing `/usr/local/share` with e.g. `/usr/share` - for projects that might prefer to have separate variables for /usr and /usr/share

This is all very confusing because different distributions have different policies and that's why I decided to give exact examples of all normal usecases I could think of.

Reply:
Ahh, thanks for the explanation - certainly don't want to add confusion or harm, if that's what it would cause!